prime-router/docs/observability/azure-events.md

```diff
@@ -93,6 +93,24 @@ Many `ReportRouteEvent` can correspond to a `ReportAcceptedEvent` and can be "jo
 - topic
     - The Topic of the report
 
+### ReportReceiverSelectedEvent
```

Comment:
I'm going to create a new event that triggers any time a report does not pass a receiver's filters.

Currently, we fire the `ReportRouteEvent` with receiver set to `null` but that's wrong because this will occur per receiver. Originally the `FHIRRouter` only fired that event when a bundle didn't get routed to _any_ receivers. I did double check that `FHIRDestinationFilter` handles that case.

Reply:
Okay so we will do the following?

`FHIRReceiverFIlter`:
- Fire `ReportRouteEvent` if report passed all receiver filters
- Fire a `new event` if report did not pass all receiver filters

`FHIRDestinationFilter`:
- Fire `ReportAcceptedEvent` for each receiver a report was sent to
- Fire a `new event` if a report did not get routed to any receiver

Reply:
One  small change:

Okay so we will do the following?

`FHIRReceiverFIlter`:
- Fire `ReportRouteEvent` if report passed all receiver filters
- Fire a `new event` if report did not pass all receiver filters

`FHIRDestinationFilter`:
- Fire `ReportAcceptedEvent` for each receiver a report was sent to
- ~~Fire a `new event` if a report did not get routed to any receiver~~
- Fire `ReportRouteEvent` with a null receiver value if a report did not get routed to any receivers